browser/ui/webui/brave_settings_ui.cc

```diff
@@ -169,6 +179,11 @@ void BraveSettingsUI::AddResources(content::WebUIDataSource* html_source,
       "verticalTabStripFeatureEnabled",
       base::FeatureList::IsEnabled(tabs::features::kBraveVerticalTabs));
 #endif
+
+#if BUILDFLAG(ENABLE_AI_CHAT)
+  html_source->AddBoolean("isLeoAssistantDisabled",
```

Comment:
We probably want to set this boolean even if AI chat is disabled (though just to false) to ensure we don't get any errors on the JS side.

Reply:
Added  the default value for `isLeoAssistantDisabled` if AI chat is disabled 
https://github.com/brave/brave-core/pull/19212/commits/a6764028640b3308007b769b8fbfba2d2649cfe0#diff-38b6c12d746e933cd876120ea21df0a3ddc37a4879dbed20f364640ad3c60bdeR187